doc/nrf/ug_zigbee_configuring.rst

```diff
@@ -19,12 +19,6 @@ Zigbee requires the following modules to properly operate in |NCS|:
 * :ref:`nrfxlib:zboss` available in nrfxlib, with the OSIF subsystem acting as the linking layer between the ZBOSS stack and |NCS|.
   OSIF implements a series of functions used by ZBOSS and is included in the |NCS|'s Zigbee subsystem.
   The files that handle the OSIF integration are located in :file:`nrf/subsys/zigbee/osif`.
-
-  * The ZBOSS stack library comes in production and development versions.
-    The production version is enabled by default with the :option:`CONFIG_ZIGBEE_LIBRARY_PRODUCTION` Kconfig option.
```

Comment:
Will this ZBOSS Zigbee stack library be enabled by default after these changes are merged?
If yes, let's keep this line: ``This library is enabled by default with the :option:`name_of_option` Kconfig option.``